src/App.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from "react";
+import BookList from "./components/BookList/BookList";
+import styles from "./App.module.scss";
+
+const App = () => {
+	//setting state for the books
+	const [booksData, setBooksData] = useState([]);
+	//setting state for the search result
+	const [search, setSearch] = useState("");
+
+	const getBooks = async (query) => {
```

Comment:
```
    const getBooks = async (query) => {
        const response = await fetch(
            `https://www.googleapis.com/books/v1/volumes?q=${query}&maxResults=12`
        );

        const data = await response.json();

        if (data.totalItems === 0) {
            setError(true);
            setBooksData([]);
        } else {
            setError(false);
            setBooksData(data.items);
        }
    };
```

if there is no books found, set the error messgae to true, if there is books, set it to false